VS/VehicleSignals.rdf

```diff
@@ -678,10 +678,12 @@
 	
 	<owl:Class rdf:about="&auto-vs;Bank2">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">bank2</rdfs:label>
```

Comment:
Could you research whether we can give it a more meaningful label>

Reply:
done